packages/app/src/app/overmind/effects/api/index.ts

```diff
@@ -538,8 +538,9 @@ export default {
       avatar,
     });
   },
-  createVercelIntegration(code: string): Promise<CurrentUser> {
-    return api.post(`/users/current_user/integrations/zeit`, {
+  // change here
```

Comment:
Can you add a comment on what needs to be changed?

Reply:
Oops, old comment. Thanks for pointing this out!